dev/py-validate.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
```

Comment:
My point of view:
- don't enforce it right away, just agree on the style
- execute manualy on some files (in my next PR, it's on documentation), and on small big bang PRs
- then enforce automatic formatting (preferably with Yapf) on some files. On Buildbot we use a git commit hook to check python files automatically, apply some pep8 formatting, execute pylint,...